lib/commands/europeana/portal/docker/docker_command.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Europeana
+  module Portal
+    class DockerCommand < Thor::Group
+      include Thor::Actions
+      include Rails::Generators::Actions
+
+      source_root File.expand_path('templates', __dir__)
+
+      argument :rails_env, type: :string, default: 'development', desc: 'Rails env'
+
+      class_option :europeana_api_key, type: :string, desc: 'Europeana API key'
+      class_option :redis_db_index, type: :numeric, default: 0, desc: 'Redis database index'
+
+      def copy_docker_compose
+        template 'docker-compose.services.yml', 'docker-compose.services.yml'
```

Comment:
why not just have the file be 'docker-compose.yml'? Then it wouldn't be necessary to add the -f flag when calling 'docker-compose xyz'

Reply:
Because of the potential use of Docker Compose to also run the app itself, which I would envisage being the default action, i.e. using `docker-compose.yml`. But as we are not doing so yet, I will rename the file as you suggest.